src/hooks/progress/use-tutorial-progress.ts

```diff
@@ -25,20 +26,30 @@ const useTutorialProgress = ({
 	tutorialId,
 	collectionId,
 }: TutorialIdCollectionId): UseTutorialProgressResult => {
-	// Get the current user's access token
+	/**
+	 * Get the current user's access token, and the batch query status.
+	 * We enable the query only if:
+	 * - an accessToken is present
+	 * - a batch query, if one exists, has been attempted
+	 */
 	const { session } = useAuthentication()
 	const accessToken = session?.accessToken
+	const hasWaitedForBatchQuery = useHasWaitedForQuery([PROGRESS_BATCH_QUERY_ID])
```

Comment:
As with `use-collection-progress`, we wait on any progress batch query (if there is one) before enabling individual tutorial progress queries.